lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -22,30 +21,175 @@ function distribute(node) {
     return distributeAndSimplifyMultiplication(node);
   }
   else if (Node.Type.isOperator(node, '^')) {
-    return expandBase(node);
+    /*
+      If it is a power node
+      1) If base is an nthRoot node, distribute the exponent
+      2) If the exponent is negative, reciprocate the base
+      3) If the base is a multiplication or power node, distribute the exponent
+      4) If the base is an addition node, expand the base
+
+      e.g. (x y)^-1 -> 1/(x y)
+      e.g. (x y)^2 -> x^2 y^2, (x^2)^3 -> x^(2 * 3)
+      e.g. (x + 1)^2 -> (x + 1) (x + 1)
+    */
+
+    const base = getContent(node.args[0]);
+    const exponent = getContent(node.args[1]);
+
+    if (Negative.isNegative(exponent)
+             || (exponent && Negative.isNegative(exponent))) {
+      return reciprocateTerm(node);
+    }
+    else if (Node.Type.isFunction(base, 'nthRoot')) {
+      return convertNthRootToExponent(node);
+    }
+    else if (Node.Type.isOperator(base, '*') || Node.Type.isOperator(base, '^')){
```

Comment:
I think I would check for these two cases separately, and call `distributeExponent` for mult and `distributeOverPowerExponent` for power. That's what's happening now anyway.